pkg/cmd/build/new.go

```diff
@@ -43,7 +46,24 @@ func NewCmdBuildNew(f *factory.Factory) *cobra.Command {
 				return fmt.Errorf("could not resolve a pipeline")
 			}
 
-			return newBuild(pipeline.Org, pipeline.Name, f, message, commit, branch, web)
+			form := huh.NewConfirm().
+				Title(fmt.Sprintf("Create new build on %s?", pipeline.Name)).
+				Affirmative("Yes!").
+				Negative("No!").
+				Value(&confirm).
+				WithTheme(huh.ThemeDracula())
+
+			if confirm {
+				form.Skip()
+			} else {
+				_ = form.Run()
+			}
+
+			if confirm {
+				return newBuild(pipeline.Org, pipeline.Name, f, message, commit, branch, web)
+			} else {
+				return errors.New("Create was aborted.")
+			}
```

Comment:
If I ctrl-c the form, it still proceeds somehow 😕

Reply:
Yeah, it does on the picker too. I think that's a tea thing.